pkg/actions/nodegroup/create.go

```diff
@@ -34,24 +32,22 @@ type CreateOpts struct {
 	ConfigFileProvided        bool
 }
 
-func (m *Manager) Create(options CreateOpts, nodegroupFilter filter.NodeGroupFilter) error {
+func (m *Manager) Create(options CreateOpts, nodegroupFilter filter.NodegroupFilter) error {
```

Comment:
nit: we tend to do `NodeGroup` instead of `Nodegroup` in the code, although we definitely aren't consistent :smile:

Reply:
I completely agree with you! The reason for it here is that the struct is `NodeGroupFilter` and the interface is `NodegroupFilter` (not very creative..)

Reply:
Maybe `NGFilter` for the struct name? Or just `Filter` for the interface?